intuita-webview/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
do you know why this is needed?

Reply:
this is needed to include react-scripts type declarations (this is autogenerated create-react-app file )